app/build.gradle

```diff
@@ -9,6 +9,9 @@ ext {
     APPLICATION_ID = "com.flatstack.android"
     isCI = "true".equals(System.getenv("CI"))
     commitMessage = 'git log -1 --pretty=%B'.execute().text.trim()
+//    PN_STORE_PASSWORD = System.getenv("PN_STORE_PASSWORD")
```

Comment:
let's rename variable name to `STORE_PASSWORD` and rename system variable key to `PROJECT_NUMBER_STORE_PASSWORD` to avoid `PN` confusing abbreviation.
Same for `KEY_ALIAS` and `KEY_PASSWORD`